docs/access.md

```diff
@@ -39,7 +39,7 @@ ARGUMENTS
   EMAIL  email address of the team member
 
 FLAGS
-  -a, --app=<value>          (required) app to run command against
+  -a, --app=<value>          (required) [default: heroku-vscode] app to run command against
```

Comment:
Watch out for `[defaults]` being added in readme files. This is due to having your git remotes connected at the time of using `oclif readme`.

Reply:
This is annoying....